src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
```

Comment:
The `href` attribute in the logo link should point to '/' for the home page instead of 'index.html'.